### PR TITLE
fix: initial readme grammar

### PR DESCRIPTION
**Pull request by rafaelramalho19**
Just quick spell checks and grammar problems I've found while reading the document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The RFC process will:
 
 ## Reviewing RFCs
 
-Members of the MOXY team will attempt to review some set of open RFC pull requests on a regular basis. If a MOXY team member believes an RFC PR is ready to be accepted, they can approve the PR using GitHub's review feature to signal their approval of the RFC.
+Members of the MOXY team will attempt to review some set of open RFC pull requests regularly. If a MOXY team member believes an RFC PR is ready to be accepted, they can approve the PR using GitHub's review feature to signal their approval of the RFC.
```
Comment (satazor) [inline README.md]: I prefer the previous wording on this phrase.

**Review by satazor — CHANGES_REQUESTED** [review]
Just a minor thing.

**Comment by satazor** [comment]
After this is done, can you port the changes to our internal RFC process?

File: README.md (hunk 1 above)
Comment (rafaelramalho19) [reply README.md]: Aight'

**Pull request merged**